src/components/tooltip/Marker.js

```diff
@@ -311,13 +311,19 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
         <TooltipDetailSeparator key="backtrace-separator" />,
         <TooltipDetail label="Stack" key="backtrace">
           <div className="tooltipDetailsBackTrace">
-            <h2 className="tooltipBackTraceTitle">
-              {data.type === 'Styles' || marker.name === 'Reflow'
-                ? `First invalidated ${formatTimestamp(
-                    causeAge
-                  )} before the flush, at:`
-                : `Triggered ${formatTimestamp(causeAge)} ago, at:`}
-            </h2>
+            {/* The cause's time might be later than the marker's start. For
+                example This happens in some usual cases when the cause is
```

Comment:
nit: `This` this should start with lowercase character. But maybe you intended to add a colon after for example and it might be missing.

Reply:
I think I just added `For example` after I wrote the full sentence and forgot to change the case of the first character.